src/pages/dev/send-tokens/interchain-tokens.mdx

```diff
@@ -3,23 +3,45 @@ import { Callout } from "/src/components/callout"
 # Interchain Tokens
 <Callout>Interchain Tokens have received a major update as of July 2023 with many new features! Read on to learn about all of the new capabilities and check out the [Interchain Portal](https://testnet.interchain.axelar.dev).</Callout>
 
-Axelar has [several ways](overview) to send [supported native tokens](/resources/testnet#assets), but what if you want have your own [ERC-20](https://ethereum.org/developers/docs/standards/tokens/erc-20/) token available on multiple chains? Axelar can help with Interchain Tokens!
+Axelar has [several ways](overview) to send [supported native tokens](/resources/testnet#assets), but what if you want to have your own [ERC-20](https://ethereum.org/developers/docs/standards/tokens/erc-20/) token available on multiple chains? Axelar can help with Interchain Tokens!
 
 Interchain Tokens are created and managed by our new  Interchain Token Service, which allows you to send tokens cross-chain, build your own asset bridges, build asset transfers into your interchain dApp, and take on many other use cases.
 
 Depending on whether or not you have an existing ERC-20 token, there are two main paths available to you.
 
-1. [I want a new Interchain Token](#new-interchain-tokens)
-1. [I want to upgrade existing token(s)](#making-existing-tokens-interchain)
+1. [Create a new Interchain Token](#create-a-new-interchain-token)
 
-## New Interchain Tokens
+    A. [Register and create a new Interchain Token using the Interchain Token Portal](#create-a-new-interchain-token) 
+    
+    B. [Create a new custom Interchain Tokens](#create-a-new-interchain-token)
+    
+2. [Upgrade an existing token(s)](#making-existing-tokens-interchain)
 
-### Standardized Tokens
-The simplest type of Interchain Token is to create a brand new [Standardized](#standardized) ERC-20 that is available on multiple chains.
+    A. [Make a token on one chain into an Interchain Token](#making-existing-tokens-interchain)
+    
+    B. [Link deployed tokens on multiple chains into Interchain Tokens](#making-existing-tokens-interchain)
```

Comment:
If you are using a sub-list, is there a standard used anywhere else in the site? I recommend we flatten this list entirely. "A" isn't a common lettering